realtime/server.js

```diff
@@ -107,16 +115,19 @@ socket.on('connection', function (client) {
     });
 
     client.on('start-typing', function(data) {
-        winston.debug('[' + people[client.id] + '] start-typing');
-
+        winston.debug('[' + people[client.id] + '] start-typing'); 
+        
+        //form object contains some info about the message which user writes
         var form = {
             username: people[client.id],
             text: data.text,
-            target: data.target,
+            target: data.target, 
             datetime_start: Date.now(),
             typing: true
         };
-
+        
+        //data.type indicates the recipient of the message(user or data)
+        //data.target indicates the name of the recipient
```

Comment:
This is documented in the architecture document in the protocol section and is unnecessary.